sdk/cognitivelanguage/Azure.AI.Language.Text/MigrationGuide.md

```diff
@@ -2043,7 +2056,7 @@ foreach (AnalyzeTextOperationResult analyzeTextLROResult in analyzeTextJobState.
         EntityRecognitionOperationResult entityRecognitionLROResult = (EntityRecognitionOperationResult)analyzeTextLROResult;
 
         // View the classifications recognized in the input documents.
-        foreach (EntitiesDocumentResultWithMetadataDetectedLanguage nerResult in entityRecognitionLROResult.Results.Documents)
+        foreach (EntitiesDocumentResultWithMetadata nerResult in entityRecognitionLROResult.Results.Documents)
```

Comment:
Can we rename this class to simply EntitiesResult and skip the document.  I think these class names are too long